x-pack/plugins/security_solution/public/management/components/console/components/command_input/key_capture.tsx

```diff
@@ -129,26 +125,14 @@ export const KeyCapture = memo<KeyCaptureProps>(({ onCapture, focusRef, onStateC
         'shiftKey',
       ]);
 
-      setLastInput((value) => {
-        onCapture({
-          value,
-          eventDetails,
-        });
-
-        return '';
+      onCapture({
+        value: newValue.length === 1 ? newValue : '',
```

Comment:
Don't know if this will work with unicode. Unicode characters usually have a length of two even for single characters (has to do with UTF-8 encoding)

<img width="268" alt="image" src="https://user-images.githubusercontent.com/56442535/187737159-bc9a5824-6330-42ac-9ccf-5db73797afbf.png">

Reply:
What kind of things does unicode support that you suspect users will need in the console?